tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = django{18,110,111}
+
+[pycodestyle]
+exclude = .git,.tox,migrations
+max-line-length = 120
+
+[testenv]
+whitelist_externals =
+    make
+deps =
+    -rrequirements/base.txt
+    -e.
+    -rrequirements/test.txt
+    django18: Django>=1.8,<1.9
+    django110: Django>=1.10,<1.11
+    django111: Django>=1.11,<1.12
+passenv =
+    DISPLAY
+commands =
+    make var/workbench.db
+    make test
+
+[testenv:quality]
+whitelist_externals =
+    make
+    rm
```

Comment:
I don't see rm used in the commands below; should it be removing the generated `__init__.py` at the end?

Reply:
Actually, the `__init__.py` file here already exists. So I'll remove the `touch` whitelisting and the `touch` command.